src/PKSim.Infrastructure/ProjectConverter/v7_3/Converter721To730.cs

```diff
@@ -44,19 +53,44 @@ public Converter721To730(ICompoundFactory compoundFactory, ICloner cloner, ICrea
 
       public void Visit(Compound compound)
       {
+         resetAllParametersToIsDefaultIn(compound);
          convertCompound(compound);
       }
 
       public void Visit(Simulation simulation)
       {
+         simulation.AllBuildingBlocks<IPKSimBuildingBlock>().Each(resetAllParametersToIsDefaultIn);
          simulation.Compounds.Each(convertCompound);
+         resetAllParametersToIsDefault(simulation.All<IParameter>());
       }
 
       public void Visit(PKSimProject project)
       {
          convertProject(project);
       }
 
+      public void Visit(IPKSimBuildingBlock buildingBlock)
+      {
+         resetAllParametersToIsDefaultIn(buildingBlock);
+      }
+
+      private void resetAllParametersToIsDefaultIn(IContainer container)
+      {
+         resetAllParametersToIsDefault(container.GetAllChildren<IParameter>());
+      }
+
+      private void resetAllParametersToIsDefault(IEnumerable<IParameter> allParameters)
+      {
+         allParameters.Each(p =>
+         {
+            p.IsDefault = true;
+            if (p.ValueDiffersFromDefault())
```

Comment:
Checks for IsFixedValue AND for value!=DefaultValue

Reply:
Where is the check for `IS_INPUT == true` done?

Reply:
The check for IS_INPUT can only be done on a case by case basis. This is implemented for Compound where I have to load a template compound and compare.

Reply:
Guess we will need this also for Formulation and Application Protocols